tuned/utils/commands.py

```diff
@@ -471,3 +471,25 @@ def save_active_profile(self, profile_name, manual):
 				f.write(mode + "\n")
 		except (OSError,IOError) as e:
 			raise TunedException("Failed to save profile mode: %s" % e.strerror)
+
```

Comment:
Is there any reason for it to be in the 'commands'? I take it as a hack for storage of utilities/helpers which could be useful in other places of the Tuned code/plugins and these two functions seems to be called just from the daemon.py.

Reply:
I wanted it to be in the same place as `save_active_profile`, etc., because those methods are related. In the future, I certainly want to move them some place else. My PR #176 already does that - later I'll update that PR to move the new methods too.

Reply:
Also, in the future they will be called from admin.py too - I'll need them to implement 6) c) and things like `tuned-adm change-post-loaded-profile`.